packages/vuetify/src/components/VBottomNavigation/VBottomNavigation.tsx

```diff
@@ -0,0 +1,107 @@
+// Styles
+import './VBottomNavigation.sass'
+
+// Composables
+import { makeBorderProps, useBorder } from '@/composables/border'
+import { makeDensityProps, useDensity } from '@/composables/density'
+import { makeElevationProps, useElevation } from '@/composables/elevation'
+import { makeLayoutItemProps, useLayoutItem } from '@/composables/layout'
+import { makePositionProps, usePosition } from '@/composables/position'
+import { makeRoundedProps, useRounded } from '@/composables/rounded'
+import { makeTagProps } from '@/composables/tag'
+import { useBackgroundColor } from '@/composables/color'
+import { useProxiedModel } from '@/composables/proxiedModel'
+
+// Utilities
+import { computed, defineComponent, toRef } from 'vue'
+import { convertToUnit } from '@/util'
+import { makeProps } from '@/util/makeProps'
+
+export default defineComponent({
+  name: 'VBottomNavigation',
+
+  props: makeProps({
+    color: String,
+    flat: Boolean,
+    grow: Boolean,
+    height: {
```

Comment:
use `size` prop from `makeLayoutItemProps` instead.

Reply:
Would this need to apply to v-app-bar as well? It implements this the same way.

Reply:
I thought you changed app-bar to use `size` prop? If you want to keep using `height`/`width` instead then you should remove `size` from `makeLayoutSizeProps`